### PR TITLE
PRINCIPLES, note about when we do and don't apply snake case

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

**Comment by wiese** [comment]
I'm confused by this as I don't quite know which "parameters" are referred to.
Is this about e.g. entityType and entityId in?

```
/entities/{entityType}/{entityId}/statements
```

If so, I consider them a non-issue to this rule, as they are not really parameters in the same sense - they are replaced in the client and their names never communicated with the server.

If not, what else is meant?

**Comment by addshore** [comment]
This is indeed the only thing this means.
The only thing that made me want to include it is because in our spec we have all of these in a single parameters.json file, and different rules apply to different types.
Perhaps it would be best to leave this out though and just aim to enforce this in CI anyway..

**Comment by wiese** [comment]
Counter proposal: #59 
Let me know if I missed the point.

File: PRINCIPLES.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ We have seen inconsistencies in previous APIs that can lead to confusion or wron
 
 Example: `entities/items/<id>/statements/<id>/qualifiers/<hash>`
 
+### Snake-case for multi-word properties
+
+We do not apply this principle to parameters included in the path.
```
Comment (itamargiv) [inline PRINCIPLES.md:36]:
```suggestion
We do not apply this principle to path variable (parameters), as they are functioning as placeholders and thus their name is not communicated to server.
```

**Review by itamargiv — COMMENTED** [review]
I would prefer just mentioning it here rather than changing it everywhere now.

**Comment by addshore** [comment]
I'm going to close this one as I think either we want to do #59, or just not bother with either.

**Pull request closed without merging**